modules/evs/fsw/src/cfe_evs_task.c

```diff
@@ -134,12 +129,9 @@ int32 CFE_EVS_EarlyInit(void)
         }
         else
         {
+            /* Convert to CFE success type */
```

Comment:
Minor nitpick - this overwrote `OS_SUCCESS` with `CFE_SUCCESS` here because one should not assume they are the same value.  To be pendantic they should probably be separate variables - we shouldn't intermix osal status and cfe status codes.

At least in this case there was a method behind the madness.